pyta.py

```diff
@@ -10,24 +10,25 @@
 """
 import importlib.util
 import pylint.lint as lint
-from pylint.reporters import BaseReporter
+from reporters.color_reporter import ColorReporter
 
 import webbrowser
 import os
 from urllib.request import pathname2url
 
+
 # Local version of website; will be updated later.
 HELP_URL = 'file:' + pathname2url(os.path.abspath('website/index.html'))
 
 
-def check(module_name):
+def check(module_name, reporter=ColorReporter()):
```

Comment:
Don't pass in an instance of the class, pass in the class itself `reporter=ColorReporter` and then call it to create a new reporter in the body of this function.

This is precisely the "mutable default argument" problem.